subprojects/gradle/test-project/src/main/kotlin/com/avito/test/gradle/module/AndroidAppModule.kt

```diff
@@ -0,0 +1,139 @@
+package com.avito.test.gradle.module
+
+import com.avito.test.gradle.buildToolsVersion
+import com.avito.test.gradle.dir
+import com.avito.test.gradle.file
+import com.avito.test.gradle.files.InstrumentationTest
+import com.avito.test.gradle.files.androidManifest
+import com.avito.test.gradle.files.build_gradle
+import com.avito.test.gradle.kotlinClass
+import com.avito.test.gradle.kotlinVersion
+import com.avito.test.gradle.module
+import com.avito.test.gradle.sdkVersion
+import java.io.File
+
+class AndroidAppModule(
+    override val name: String,
+    override val packageName: String = "com.$name",
+    override val plugins: List<String> = emptyList(),
+    override val buildGradleExtra: String = "",
+    override val modules: List<Module> = emptyList(),
+    override val enableKotlinAndroidPlugin: Boolean = true,
```

Comment:
How did you choose a default value? Is it safer?

I'm asking because of multiple `enableKotlinAndroidPlugin = false` in clients. 
What will be a trigger for me to disable it?

Reply:
It's created only for ignoring `kotlin` and `android` Gradle plugins exceptions

Reply:
We want to apply the `kotlin` plugin always. But I said we have configuration errors when your module have kotlin or android plugins in the classpath.
And do not apply the kotlin plugin is a workaround I found